### DISCUSSION TITLE
Can I change the topic mqtt?

### DISCUSSION THREAD

**Discussion by jaytnw** [jblance/mpp-solar#80] [Q&A]
I wanna change topic from '#' to any topic. help me please.

**Comment by jblance** [comment]
# literally means any topic.
What are you trying to do - nothing in mpp-solar should be posting to such a topic name

**Reply by jaytnw** [reply]
I want to change topic for publish from default topic ('#') to my topic i.e. 'test/solar'. I can't use parameter --mqtttopic

**Comment by jblance** [comment]
What command and output module are you using? What does ```mosquito_sub -v -t "#"``` give?

--mqtttopic is new and only implemented for one output module so far, the others derive the topic in different ways (noting the the topic will not be # but something specific, # is just a way to see all topics
